mne/gui/_kit2fiff_gui.py

```diff
@@ -46,8 +47,8 @@
     hsp_wildcard = ['Head Shape Points (*.hsp;*.txt)|*.hsp;*.txt']
     elp_wildcard = ['Head Shape Fiducials (*.elp;*.txt)|*.elp;*.txt']
     kit_con_wildcard = ['Continuous KIT Files (*.sqd;*.con)|*.sqd;*.con']
-elif os.name == 'nt':
-    # on Windows, multiple wildcards does not seem to work
+elif sys.platform in ('win32',  'linux2'):
```

Comment:
Why don't we just always do it the slightly longer but always-working way? It's less branching